content/en/developers/metrics/_index.md

```diff
@@ -17,20 +18,28 @@ further_reading:
   text: "Official and Community created API and DogStatsD client libraries"
 ---
 
-A Datadog metric is defined by the properties below. Refer to the [Metrics Introduction documentation][1] to learn how to graph metrics within Datadog.
+## Overview
+
+If a metric is not submitted from one of the [350+ Datadog integrations][1] it's considered a custom metric (with [some exception][2]).
+
+A custom metric refers to **a unique combination of metric name, host, and tag values**. In general, any metric you send using [DogStatsD][3], or through a [custom Agent Check][4] is a custom metric. There are no enforced fixed rate limits on custom metric submission. If your default allotment is exceeded, you are billed according to [Datadog's billing policy for custom metrics][5].
```

Comment:
With the introduction of metrics without Limits, as well as custom metrics from "Metrics from Logs" and Lambda metrics, we're seeing more and more that custom metrics don't necessarily have to be uniquely identified as follows  "A custom metric is identified by a unique combination of a metric's name and tag values (including the host tag). " since they'd lack a "host" tag

That's why I'd rather have this wording: "A custom metric is identified by **a unique combination of a metric's name and tag values (including the host tag)**". This conveys that a metric name + tag values is still a custom metric even if host:blah isn't in that set of tag values